### ISSUE TITLE
Introduction to Neural Network: Backpropagation

### ISSUE THREAD

**Issue by Kulbear** [Kulbear/Schedule#3]
## Refresher: The Gradient Descent
The **gradient descent** is one of the simplest methods we can apply to multiple algorithms for an optimization purpose.  

Frist of all, you probably realize that we need a metric to measure our prediction error. There are many metrics we can choose, such as **mean squared error (MSE)**, **mean absolute error (MAE)**, **R2-score** and so on. It would be another story if you want to dive into those concepts and have a clear understanding which one is best in a specific case. 

Google and Wikipedia are always your best friends. Beside from these resources, I'd like to share a post on [StackExchange](http://stats.stackexchange.com/a/48268) if you want a quick shot about how to select a better metric for your case from **MSE** and **MAE**.
